src/fields/MultiplicityField.jsx

```diff
@@ -65,7 +65,7 @@ class MultiplicityField extends React.Component {
     values[position] = newvalue;
     this.setStateAsChange({ values }, () => {
       if (this.props.onUpdate) {
-        this.props.onUpdate(e, name, field, values);
+        this.props.onUpdate(e, name, field, values.filter(val => !!val.trim()));
```

Comment:
the !! should not be necessary here JS should  already perform that coercion because filter needs a boolean as function output